app/Models/Task.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Task extends Model
+{
+    protected $table = 'tasks';
+    protected $fillable = [
```

Comment:
biến này dùng để làm gì?

Reply:
mỗi một model sẽ ứng với một bảng, biến $table để đặt gán tên bảng trong database cho model viết theo kiểu snake_case, biến $fillable để liệt kê các cột sẽ được hiển thị, nêus không có sẽ ko hiển thị ra gì. Trong trường hợp ko có cột created_at và updated_at trong database thì timestamps để giá trị là false ko thì bị lỗi ạ.

Reply:
> biến $fillable để liệt kê các cột sẽ được hiển thị, nêus không có sẽ ko hiển thị ra gì

giải thích lại phần này nào

Reply:
biến $table để đặt tên bảng mà model ánh xạ tới, mảng $fillable để lấy các cột sẽ hiển thị ạ

Reply:
trong mang $fillable chua cac field ma chung ta muon kiem soat khi thay đoi gia tri, bo vao bien array $fillable đe bao cho Laravel biet la cac field nay co the thay đoi gia tri boi nguoi dung.

Reply:
> trong mang $fillable chua cac field ma chung ta muon kiem soat khi thay đoi gia tri, bo vao bien array $fillable đe bao cho Laravel biet la cac field nay co the thay đoi gia tri boi nguoi dung.

đọc thêm về mass assignment nhé